### PR TITLE
Adds Localization into the whole backend.

### PR DESCRIPTION
**Pull request by masadmunawar**
Adds Localization into the whole backend. and updates the APIs names by adding country Code

### CODE REVIEW

File: src/routers/daignonsisResult.js [hunk 1]
```diff
@@ -91,11 +91,23 @@ router.get("/calculateDiagnotics", async (req, res) => {
       // Round the percentage to the nearest whole number and add it to the resultPercentage array
       let percentage = Math.round(per + probability);
       per = 0;
-      let obj = {
-        possibleDiagnostic: populateDiagnosis[i].diagnosticsId.diagnosisName,
-        percentage: percentage
-      };
-      resultPercentage.push(obj);
+      if (req.params.countryCode == "es") {
```
Comment (anees-rehman-techtimize) [inline src/routers/daignonsisResult.js:94]: define an Enum for Country code, and use the enum key in If conditions. avoid the hardcoded string "es" and "en"

File: src/routers/daignonsisResult.js [hunk 2]
```diff
@@ -91,11 +91,23 @@ router.get("/calculateDiagnotics", async (req, res) => {
...
+        let obj = {
+          possibleDiagnostic: populateDiagnosis[i].diagnosticsId.diagnosisNameEs,
+          percentage: percentage
+        };
+        resultPercentage.push(obj);
```
Comment (anees-rehman-techtimize) [inline src/routers/daignonsisResult.js:99]: Code is duplicated in if and else block, create the obj before if, and add the obj in resultPercentage at the end of IF condition

**Review by anees-rehman-techtimize — CHANGES_REQUESTED** [review]
Please create an Enum for country codes and use it in If conditions. Also try to minimize the code duplication in each if-else code blocks.

File: src/routers/daignonsisResult.js (hunk 1 above)
Comment (masadmunawar) [reply src/routers/daignonsisResult.js]: Done

File: src/routers/daignonsisResult.js (hunk 2 above)
Comment (masadmunawar) [reply src/routers/daignonsisResult.js]: Done

**Review by anees-rehman-techtimize — APPROVED** [review]

**Pull request merged**